src/util-logopenfile.c

```diff
@@ -785,7 +785,7 @@ static bool LogFileThreadedName(
          * for update
          */
         dot = strrchr(original_name, '.');
-        int dotpos = dot - original_name;
+        long dotpos = dot - original_name;
```

Comment:
is this meant to be a type like `ptrsize_t`?

Reply:
`ptrsize_t`does not exist on my Mac, going for `ptrdiff_t`